Chore: specify more loose options for babel

fatfisz opened the pull request:
Rollup is great: it reduces the number of `require` calls and produces a clean output, while putting all of the modules in the same scope.

I've also set the "loose" option for babel plugins for a cleaner output.

tleunen commented:
I don't really see the benefit of using rollup here. This is strictly a node module, not for the browser so there's no reason to bundle it

fatfisz commented:
The benefits are: small IO performance gain because of only one require and more readable output (babel adds a lot this "interop" code which is a bit annoying when debugging). If you feel that it's not worth it, then feel free to close this without merging; I just think of it as a nice-to-have, since there is a compilation step anyway.

tleunen commented:
Yeah, I'd prefer removing rollup. But keeping the `babel-preset-env` config with node 4 is ok.

Also, you can use the `useBuiltIns` option, and `loose` as well. It should make the output a bit more readable if you want :)

fatfisz commented:
@tleunen I've updated the branch and changed the name of the issue to reflect the changes. `_interopRequireDefault` stays, but at least no more `toConsumableArray` :)
